pyhanko/cli/commands/signing/__init__.py

```diff
@@ -33,6 +33,12 @@ def signing():
 @click.option('--name', help='explicitly specify signer name', required=False)
 @click.option('--reason', help='reason for signing', required=False)
 @click.option('--location', help='location of signing', required=False)
+@click.option('--contact-info', help='contact of the signer', required=False)
+@click.option(
```

Comment:
Is this a sensible option to expose in the CLI? I don't think it ever applies in a command-line scenario, to be honest.

Reply:
Hi, @MatthiasValvekens! I'm so sorry for the delay in answering. I had been in problems with time and with my computer on last weeks. Thank you so much for your feedback.

About your comment here, it makes sense. But what do you mean with `I don't think it ever applies in a command-line scenario` ?. What would be the differences using this option in API and in CLI? I'm not so exeprient with lib's development, so it would be really important to understand your point.

Reply:
@eduperottoni if you agree with this change I'll apply it myself, then rebase & merge your contribution.

Reply:
Perfect, Matthias! Your point completely understood and I agree, makes all sense! 

I removed `prop_auth_time` option from CLI and added a test in CLI tests including those signature's entries options.